pages/dkp/kommander/2.2/operations/access-control/index.md

```diff
@@ -139,18 +121,31 @@ subjects:
     name: oidc:engineering
 ```
 
-### Role Bindings
+## Role Bindings
+
+Kommander role bindings, cluster role bindings, and project role bindings bind a group of users to any number of roles. All groups defined by [Identity Providers][groups] are present in the **Groups** tab at the global, workspace, or project level.
```

Comment:
the "Groups" tab only exists at the global level for Identity Providers.

Everywhere else the tabs are Role Binding tabs. 

Role bindings or Cluster Role Binding tabs show a table with all the groups and all user to bind roles to those groups.

Reply:
Yes - something like: 

```suggestion
Kommander role bindings, cluster role bindings, and project role bindings bind a group of users to any number of roles. All groups defined by [Identity Providers][groups] are present in the **Role Bindings** tab at the global and workspace level in the Access Control page, and in **Role Bindings** tab in the project level.
```